src/main/java/ch/epfl/biop/operetta/commands/OperettaImporterInteractive.java

```diff
@@ -176,6 +186,13 @@ private String getMessage(long bytes_in, long bytes_out, String name, String ori
 
     private void updateMessage() {
         try {
+
+            // Choose flip mode
+            FLIP_MODE flip = FLIP_MODE.NONE;
+            for ( FLIP_MODE f : FLIP_MODE.values() ) {
+                if (f.getName().equals(this.flip_mode)){ flip = f; }
+            }
```

Comment:
This could be much simpler if you use an enum for `flip_mode`, see comment above.